scvi/dataset/retina.py

```diff
@@ -14,4 +14,5 @@ def __init__(self, save_path='data/'):
             self.download()
 
         print("Finished preprocessing Retina dataset")
-        super(RetinaDataset, self).__init__(filename=self.download_name, save_path=self.save_path)
+        super(RetinaDataset, self).__init__(filename=self.download_name, save_path=self.save_path,
+                                            batch_col="Batch_id", cluster_col="Clusters")
```

Comment:
Can we change the field name "Batch_id" (in `retina.loom`) to whatever convention Sten uses in `cortex.loom`, rather than allowing it to vary?

Or have you seen loom files (other than our `retina.loom` dataset) that refer to `batch_col` and `cluster_col` is multiple ways? What do they call them besides "Batch_id` and "Clusters"?

Reply:
One thing I thought of is, for `FISH` data, two columns can be used for `labels`: `ClusterID` and `ClusterName`. 
Also, when users create the `.loom` files, they can specify column and row names. It might be a problem when people use small variations, like `clusterID`, `cluster_id`, or `Cluster_ID`, which might crush the code. 
But I can change the `retina.loom` and remove the parameters if that's needed ^ ^

Reply:
Do `ClusterID` and `ClusterName` specify different clustering, or is one just an integer and one a string? If the former, let's follow the conventions in the existing loom data for `retina.loom` --- `ClusterID` rather than `Clusters`. And then assume for now that loom data will always use `ClusterID` for the cluster column, so that you don't have to pass `cluster_col` around all over the place.  (Same for `batch_col`.)

Reply:
They are the same - one is numbers the other is string. Sounds good! I'll make these changes.